test/verify/check-subscriptions

```diff
@@ -47,6 +47,17 @@ import os
 # register with: activation key "awesome_os_pool" and org "admin"
 # or: subscription-manager register --activationkey awesome_os_pool --org admin --serverurl https://$IP:8443/candlepin
 
+WAIT_SCRIPT = """
+set -ex
+for x in $(seq 1 20); do
+    if curl --insecure -s https://%(addr)s:8443/candlepin; then
+        break
+    else
+        sleep $x
```

Comment:
We back off the same way in check-realms. And nslookup there is also rather fast. We want the entire thing to start off polling quickly and then poll slower over time. And this pattern has worked quite well for that.